tensorrt_llm/_torch/pyexecutor/resource_manager.py

```
@@ -709,8 +730,13 @@ def calculate_cache_size_per_token(layers: Set[int]) -> int:
         for window_size in sorted(window_size_to_layers):
             layers = window_size_to_layers[window_size]
             cache_size_per_token = calculate_cache_size_per_token(layers)
-            cache_size_bytes_per_token = cache_size_per_token * binding_dtype_size(
-                dtype)
+            cache_size_bytes_per_token = get_size_in_bytes(
+                cache_size_per_token, dtype)
+            if dtype == DataType.NVFP4:
+                cache_size_bytes_per_token += self.calculate_scaling_factor_size_bytes(
```

Ruff (0.12.2)

Undefined name `self`

(F821)